src/API/src/config/config.controller.ts

```diff
@@ -80,8 +81,21 @@ fs.watchFile(`${config.get('configFolder')}/species_list.json`, () => {
 
 @Controller('config')
 export class ConfigController {
+  constructor(private mailerService: MailerService) {}
   @Get('featureflags')
   async getFeatureFlags(): Promise<{ flag: string; on: boolean }[]> {
+    this.mailerService.sendMail({
```

Comment:
It's good that you've tried this - I wasn't expecting it to be in the real code though. I'm not sure you want an email every time we get the feature flags.